editoast/src/models/errors/mod.rs

```diff
@@ -168,6 +170,14 @@ impl InfraError {
             },
         }
     }
+
+    fn new_no_buffer_stop(field: String) -> Self {
+        Self {
+            field,
+            is_warning: true,
+            sub_type: InfraErrorType::NoBufferStop,
+        }
+    }
```

Comment:
```suggestion
    fn new_no_buffer_stop<T: AsRef<str>>(field: T) -> Self {
        Self {
            field.as_ref().into(),
            is_warning: true,
            sub_type: InfraErrorType::NoBufferStop,
        }
    }
```

We could do the same with all of the Error constructors.